docker-compose.yml

```diff
@@ -90,7 +90,15 @@ services:
       - '1025:1025'
       - '8025:8025'
     image: mailhog/mailhog
-    
+  wiremock:
```

Comment:
What do you think about putting Wiremock in `docker-compose.cypress.yml`?

Reply:
hmm I would have to rewrite the nginx config since it depends on the wiremock container. I added a wiremock nginx route https://localhost.simplereport.gov/wiremock

Reply:
Done!